extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1418,6 +1421,23 @@ export class AmpStoryPage extends AMP.BaseElement {
     );
   }
 
+  /**
+   * Checks if the page has playable elements.
+   * @private
+   */
+  checkPageHasPlayable_() {
+    const pageHasPlayable =
+      this.element.hasAttribute('background-audio') ||
+      this.element.querySelector('amp-audio') ||
+      this.element.querySelector('amp-video') ||
+      this.element.hasAttribute('auto-advance-after');
```

Comment:
could you move the `hasAttribute` to the left-most part of this evaluation? JS will short-circuit it when it finds a `true`, and since checking the attribute is cheaper than doing a `querySelector` we want to have those first.

https://codeburst.io/javascript-what-is-short-circuit-evaluation-ff22b2f5608c